go/filter_client/filter_client.go

```diff
@@ -0,0 +1,117 @@
+package filter_client
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"net"
+)
+
+const default_socket_path string = "/usr/local/pf/var/run/pffilter.sock"
```

Comment:
No, I believe we currently just refer to the known Golang standards for now

Unless @louismunro had fun doing it